utils/fscryptctl/Makefile

```diff
@@ -0,0 +1,51 @@
+include $(TOPDIR)/rules.mk
```

Comment:
Does this Makefile need a license, similar to other package Makefiles in this repo? (SPDX line)